src/components/MapComponent.tsx

```
@@ -1,55 +1,89 @@
-export default MapComponent;
+import React from "react";
+import { contactConfig } from "../config/contact";
+import { useTranslation } from "react-i18next";
+import { MapContainer, TileLayer, Marker, Popup } from 'react-leaflet';
+import L from 'leaflet';
+import markerIcon from 'leaflet/dist/images/marker-icon.png';
+import markerIcon2x from 'leaflet/dist/images/marker-icon-2x.png';
+import markerShadow from 'leaflet/dist/images/marker-shadow.png';
+
+// Fix for default markers not showing in Vite/Webpack builds
+delete (L.Icon.Default.prototype as any)._getIconUrl;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)